### DISCUSSION TITLE
Docker container, not recognizing linknet_resnet18_rotation

### DISCUSSION THREAD

**Discussion by orlandito24** [mindee/doctr#1394] [Q&A]
I'm trying to run a docker container, containing a file that has a model with ocr_predictor and linknet_resnet18_rotation. But I'm getting an error that linknet_resnet18_rotation is an unknown architecture.

**Comment by felixdittrich92** [comment]
Hi @orlandito24 👋 linknet_resnet18_rotation is removed instead you can use linknet_resnet18 now (works with rotations)
